### PR TITLE
Jacob angulo

### PR DESCRIPTION
**Pull request by jacobAngulo**
# Description

fixed landing page tiles from overflowing, tweaked landing page jsx, added landing page mobile styles.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/scss/Landing.scss [hunk 1]
```diff
@@ -39,37 +53,37 @@ $mobile: 500px;
...
-          color: #e8ebef;
-          width: 180px;
-          height: 60px;
-          background: #2d90f5;
-          border: 1px solid #2d90f5;
-          border-radius: 5px;
-          cursor: pointer;
-        }
-      }
+      // .mobile-buttons {
```
Comment (workingjubilee) [inline src/scss/Landing.scss]: I personally would go even further here... just rip it out. It's still gonna be there in version history if we need to recover it. And it looks stable on https://dev.rxid.net

File: src/scss/Landing.scss [hunk 2]
```diff
@@ -79,7 +93,12 @@ $mobile: 500px;
       margin: 150px 100px 50px 50px;
       line-height: 100%;
       @media screen and (max-width: $mobile) {
-        display: none;
+        height: 25%;
+        flex-direction: column;
+        justify-content: space-between;
+        width: 100%;
+        margin: 1.5rem 1rem 0.5rem 0.5rem;
+        line-height: 100%;
```
Comment (workingjubilee) [inline src/scss/Landing.scss]: Nice.

File: src/scss/Landing.scss [hunk 3]
```diff
@@ -129,14 +158,22 @@ $mobile: 500px;
...
+        @media screen and (max-width: $mobile) {
+          display: none;
+        }
       }
     }
 
     .tiles {
       display: flex;
       width: 50%;
+      margin-left: -35px;
```
Comment (workingjubilee) [inline src/scss/Landing.scss]: I slightly dislike having negative margin adjustments, it just feels "wrong", but I'm not seeing a great alternative right now, and the improvement is there, so.

File: src/scss/Landing.scss [hunk 4]
```diff
@@ -230,6 +284,10 @@ $mobile: 500px;
         font-size: 20px;
         font-weight: 300;
         color: #74869b;
+
+        @media screen and (max-width: $mobile) {
+          display: none;
+        }
```
Comment (workingjubilee) [inline src/scss/Landing.scss]: Thank you for taking the time to iterate through all these mobile edge-cases.

File: src/scss/Landing.scss (hunk 2 above)
Comment (jacobAngulo) [reply src/scss/Landing.scss]: I done did it

File: src/scss/Landing.scss (hunk 4 above)
Comment (jacobAngulo) [reply src/scss/Landing.scss]: Holy moly this was a while ago, will merge now, sorry for the delay.

File: src/scss/Landing.scss (hunk 3 above)
Comment (jacobAngulo) [reply src/scss/Landing.scss]: Same and same.

**Pull request merged**